### ISSUE TITLE
Beta4 Bust-A-Move / Puzzle Bobble (set1) broken

### ISSUE THREAD

**Issue by Jeff-Bootonic** [krabobmkd/amigamame#19]
The game starts, but the screen remains black except for the status bar. 
No matter which BIOS is used, whether official or UniBIOS, the result is the same.

Great progress, thank you!

**Comment by krabobmkd** [comment]
oO
Looks like it has been corrected in R1 , which is out... just now.
Though, I've tested "pbubble", there is a "taito_b" arcade version and a neogeo version of this same game.
In last ressort, use the other ;)

 No seriously , a lots of 68000 games were broken on beta4 and repaired in R1. That's because I'm testing the best comprise on speed and compatibility with memory system and its 68000 interface. 

 Now only "1944" is broken, and Bubble Bobble2 crash as soon as a boss is touched. Certainly because of rare memory access or GCC compiler aliasing with musashi.
